pyalgotrade/barfeed/csvfeed.py

```diff
@@ -143,7 +143,10 @@ def __init__(self, columnNames, dateTimeFormat, dailyBarTime, frequency, timezon
         self.__columnNames = columnNames
 
     def _parseDate(self, dateString):
-        ret = datetime.datetime.strptime(dateString, self.__dateTimeFormat)
+        if self.__dateTimeFormat == "%s":
```

Comment:
Instead of using %s I'd recommend using %d since that column is expected to have an integer. On top of that, please update the doc for *setDateTimeFormat*